packages/shared/src/components/cards/common/Card.module.css

```diff
@@ -3,7 +3,7 @@
 }
 
 .card {
-  & > * {
+  &:not(:global(.clickable)) > * {
     pointer-events: none;
   }
```

Comment:
Could not reuse the `<Card>` component because was preventing the tooltip to show on hover